File: patterns/java/Pyramid.java
public class Pyramid {
    public static void pyramid(int n) {
        for (int i = 1; i <= n; i++) {
            System.out.print(" ".repeat(n - i));
            System.out.println("*".repeat(2 * i - 1));
        }
    }
    public static void main(String[] args) {
        pyramid(5);
    }
}

File: patterns/csharp/Intermediate/Pyramid.cs
using System;

class Pyramid {
    static void pyramid(int n = 5) {
        for (int i = 1; i <= n; i++) {
            Console.Write(new string(' ', n - i));
            Console.WriteLine(new string('*', 2*i - 1));
        }
    }
    static void Main() {
        pyramid(5);
    }
}
